db/migrate/20250619163228_create_potential_duplicate_persons.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreatePotentialDuplicatePersons < ActiveRecord::Migration[7.2]
+  def change
+    create_table :potential_duplicate_persons do |t|
+      t.references :duplicate_checker_job_run, null: false, foreign_key: { to_table: :duplicate_checker_job_runs }
```

Comment:
I think in this specific case (name of the reference is exactly the table name in plural) Rails _should_ be able to infer the `to_table` automatically and you can just set `foreign_key: true`